discovery-kubernetes-api/src/main/scala/akka/discovery/kubernetes/PodList.scala

```diff
@@ -4,17 +4,41 @@
 
 package akka.discovery.kubernetes
 
-import scala.collection.immutable.Seq
+import java.util.Optional
```

Comment:
All of this can be internal I think?

Reply:
ok, looks like it's only used in JsonFormat. Is there a reason why we "always" make JsonFormat classes public?
if we make this internal we must make the JsonFormat internal

Reply:
It is useful for anyone wanting to consume the APIs programatically in the case of the Akka management formats but this one is for interacting with the k8s API so if it can be private it I think we should make it so

Reply:
done in https://github.com/akka/akka-management/pull/436